apps/imdhub-public/src/components/AppFooter.vue

```diff
@@ -32,12 +32,12 @@
         <div class="footer-column footer-logos">
           <UnorderedList listType="none">
             <li id="project-logo-link">
-              <router-link :to="{ name: 'home' }">
+              <a href="http://recon4imd.org">
                 <img
                   src="/recon4imd.png"
                   alt="Reconstruction and Computational Modelling for Inherited Metabolic Diseases"
                 />
-              </router-link>
+              </a>
             </li>
             <li>
               <a href="https://www.molgenis.org">
```

Comment:
you removed the citation requirements. I am happy to provide a shorter version of the standard footer so it fits on one line but I want some citation requirement in there.